msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerMsalController.java

```diff
@@ -555,6 +557,16 @@ public void putValueInSuccessEvent(ApiEndEvent event, Boolean result) {
                 });
     }
 
+    // Placeholder methods to avoid inheritance error (these methods will be declared in BaseController)
+    // Also why the @Override annotation is missing
+    public AuthorizationResult deviceCodeFlowAuthRequest(DeviceCodeFlowCommandParameters parameters) throws ClientException {
+        throw new ClientException("deviceCodeFlowAuthRequest() not supported in BrokerMsalController");
+    }
+
+    public AcquireTokenResult acquireDeviceCodeFlowToken(AuthorizationResult authorizationResult) throws ClientException {
+        throw new ClientException("deviceCodeFlowAuthRequest() not supported in BrokerMsalController");
+    }
```

Comment:
When will these be declared in `BaseController`? These would need the `@override` annotation when that happens. I suggest opening a PR for common now to take care of that and then add the annotation here so that we don't have to open a separate PR later to address the annotation.

Reply:
Also JFYI, you will probably need to open a PR for broker (ad-accounts) as well to do the same in `BrokerLocalController`

Reply:
Yes The broker code is pushed, just need to make the PR

Reply:
My timeline for the PR's was complete this one, then send out a Broker PR for `BrokerLocalController`, and then a PR to common for adding the methods to `BaseController`

Reply:
I don't need to wait for this PR to be complete to open the broker PR. But I will need this PR and the broker PR to be complete before I can complete the next Common PR.

Reply:
I'm confused why it is like that. It should be the other way around i.e. you need the Common PR to be complete before you can complete this PR and the Broker PR.

Reply:
If I declare new abstract methods to `BaseController` in Common, I'll need to implement them in the controller classes that extend `BaseController`. Wouldn't this mean that until the PRs for MSAL and Broker that implement these methods are pushed, the controller classes will have compilation errors? I'm trying to avoid these errors by pushing the methods in `LocalMsalController` and other classes first, before declaring in `BaseController`.